Kitodo/src/main/webapp/WEB-INF/resources/js/filter.js

```diff
@@ -0,0 +1,16 @@
+function setFilter(filterString) {
+    document.getElementById('filterMenu:filterfield').value = filterString;
+    applyFilter(filterString);
+}
+
+function applyFilter(filterString) {
+    var invisibleFilter = document.getElementById('j_id_2q:j_id_2s:j_id_2t:j_id_2u:filter');
```

Comment:
Is it possible to use a clear name for this filter element instead of `j_id_2q:j_id_2s:j_id_2t:j_id_2u:filter`?